sources/packages/web/src/views/aest/CASInvoices.vue

```diff
@@ -68,14 +68,14 @@
               {{ dateOnlyLongString(item.approvalStatusUpdatedOn) }}
             </template>
             <template #[`item.actions`]="{ item }">
-              <!-- TODO: change role -->
-              <check-permission-role :role="Role.AESTEditCASSupplierInfo">
+              <check-permission-role :role="Role.AESTCASInvoicing">
```

Comment:
Minor. Is the check permission not redundant here as the side-nav is already having the check?

Reply:
The link is disabled in the left menu, but the route is still available to be accessed since we do not apply the "page permission" that we apply to the institutions. Does it make sense?
Since the API should block the GET and no data should be displayed, I am ok to remove it if it will generate any noise in the PR.